docs/en/observability/add-observability-data.asciidoc

```diff
@@ -12,30 +12,30 @@ You can use our
 https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}]
 on {ecloud} (recommended), or self-manage the {stack} on your own hardware.
 
-include::tab-widgets/deploy-ess-stack-widget.asciidoc[]
+include::./../shared/spin-up-the-stack/widget.asciidoc[]
```

Comment:
I've started using an attribute called `{tab-widgets}` to resolve the path (so you can move things around without having to update relative links). Would also let us add a shared attributes file to define the location for all projects. Just build the path starting with a built-in attribute (or an attribute based on one that's built in), like `{observability-docs-root}`, so that it works from anywhere.

Reply:
Brilliant